guides/common/modules/con_prerequisites-for-network-boot-provisioning.adoc

```diff
@@ -0,0 +1,16 @@
+[id="prerequisites-for-network-boot-provisioning"]
+= Prerequisites for network-boot provisioning
+
+The requirements for network-boot provisioning include:
+
+* A {SmartProxyServer} managing the network for the network-booted machines.
+For unattended provisioning and discovery-based provisioning, {ProjectServer} requires PXE server settings.
```

Comment:
This is unclear to me. I'd probably use "SmartProxy" in line 6 (so that it works for Foreman Server and a Smart Proxy Server) and use "Project" in line 7.